avocado/plugins/spawners/podman.py

```diff
@@ -192,7 +192,10 @@ async def _create_container_for_task(
             runtime_task.task.status_services[0].uri = mounted_status_server_socket
 
         _, _, python_binary = await self.python_version
-        entry_point_args = [python_binary, "-m", "avocado.core.nrunner", "task-run"]
+
+        module_name = runtime_task.task.runnable.kind.replace("-", "_")
+        full_module_name = f"avocado.plugins.runners.{module_name}"
+        entry_point_args = [python_binary, "-m", full_module_name, "task-run"]
```

Comment:
@clebergnu @richtja How is this supposed to work with the Avocado VT runner? I want to update the LXC spawner pull request but we actually run VT tests inside of the LXC containers. The current approach will not work and will generate
```
/usr/bin/python: No module named avocado.plugins.runners.avocado_vt
```
and to make it work I have to explicitly use
```python
full_module_name = full_nonbogus_name = "avocado_vt.plugins.vt_runner"
```
so that the VT runner becomes accessible again. The auto-guessing above has a good motivation and we would all prefer to have the runner figured out from the runnable kind, however the logic so far doesn't generalize to the VT runner where we typically create `Runnable('avocado-vt', uri, **vt_params)`.

Reply:
Hi @pevogam thanks for spotting this, I created #5588 so we can track it.